apps/src/blockly/addons/plusMinusBlocks/advancedProcedures.js

```diff
@@ -476,10 +478,11 @@ const procedureDefMutator = {
    * block.
    * @param {string} name The name of the argument.
    * @param {string} argId The UUID of the argument (different from var ID).
+   * @param {Blockly.VariableModel} variable The variable model for the parameter
    * @this {GoogleBlockly.Block}
    * @private
    */
-  addVarInput_: function (name, argId) {
+  addVarInput_: function (name, argId, variable) {
```

Comment:
nit: name this parameter `variableModel` instead?

Reply:
Oh yes, good call.